README.md

```diff
@@ -1,4 +1,4 @@
-This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.com) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://paypal.github.io/InnerSourceCommons/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
+This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.org) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://paypal.github.io/InnerSourceCommons/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
```

Comment:
Innersourcecommons.org currently does a redirect to the Paypal repo, could you change the ```organizations who apply InnerSource methodologies``` link to use the redirect there? 

We might change things around there later, such that a second round of search&replace might be warranted but that would then just remove a part of the URL not change it entirely.

Reply:
👍 good catch. updated 😁 
I couldn't find any other references to paypal.github.io...